tests/test_octodns_provider_dyn.py

```diff
@@ -932,14 +932,14 @@ def test_mod_geo_create(self, mock):
         provider = DynProvider('test', 'cust', 'user', 'pass',
                                traffic_directors_enabled=True)
 
-        # will be tested seperately
+        # will be tested seperatelyly
```

Comment:
`separately` would be the correct spelling for this one.

Reply:
This was me trying to do a fix by hand instead of letting my browser suggest the spelling fix. Doh.